doc/system-telemetry/process-docker-stats.md

```diff
@@ -0,0 +1,108 @@
+# Process and docker stats availability via telemetry agent
+
+## Revision
+
+| Rev | Date     | Author      | Change Description |
+|:---:|:--------:|:-----------:|--------------------|
+| 0.1 | 09/12/19 | Pradnya Mohite | Initial version    |
```

Comment:
please add an new revision number for the date of today. Then mentioned in the code that it is against when revision number. For further comments, you can address in the future code change not the existing one.

Reply:
@hui lets discuss this offline

Reply:
I think it's fine leaving this as-is, because this is technically still the initial version , because the PR has not yet been merged.